### PR TITLE
feat: Avatar component

### PR DESCRIPTION
**Pull request by denkristoffer**
# Purpose of PR

`Avatar` component based on the `Image` component.

### CODE REVIEW

File: packages/website/components/Playground/SandpackRenderer.tsx [hunk 1]
```diff
@@ -95,6 +95,7 @@ export function SandpackRenderer({
           'react-dom': '^17.0.0',
           'react-scripts': '^4.0.0',
           '@contentful/f36-components': '^4.0.0',
+          '@contentful/f36-avatar': '^4.0.0', // Remove when added to f36-components
```
Comment (massao) [inline packages/website/components/Playground/SandpackRenderer.tsx:98]: Doesn't it need the `-alpha.0`??

**Review by mshaaban0 — APPROVED** [review]
Some minor comments, other than that. Looks cool 💅🏼

File: packages/components/avatar/README.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+In some cases a user or an app will have no avatar to display. Instead, a fallback can be rendered when the `src` property is undefined (or an empty string `""`) and the loading state is not `true`.
+
+```jsx file=./examples/AvatarFallbackExample.tsx
+
+```
+
+### Loading skeleton
+
+When the avatar image is loading, a loading skeleton will be shown automatically. You can also force the loading state with the `isLoading` prop.
```
[the remark this answers is not in the document]
Comment (denkristoffer) [reply packages/components/avatar/README.mdx:55]: Yeah it's an intentional decision by @fabe 😄

File: packages/components/avatar/src/Avatar.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  Large = '48px',
+}
+
+export enum Variant {
+  App = 'app',
+  User = 'user',
+}
+
+export interface AvatarProps extends CommonProps {
+  alt?: ImageProps['src'];
```
[the remark this answers is not in the document]
Comment (denkristoffer) [reply packages/components/avatar/src/Avatar.tsx]: On second thought I agree with that.

File: packages/components/avatar/src/Avatar.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  isLoading?: boolean;
+  size?: Size;
+  src?: ImageProps['src'];
```
[the remark this answers is not in the document]
Comment (denkristoffer) [reply packages/components/avatar/src/Avatar.tsx:23]: How would you know to render the fallback then?

[the remark this answers is not in the document]
Comment (denkristoffer) [reply packages/components/avatar/src/Avatar.tsx:23]: I'm merging but we can revisit this decision in a new version.

[the remark this answers is not in the document]
Comment (mshaaban0) [reply packages/components/avatar/src/Avatar.tsx:23]: You're right, sorry I missed that.
However I think fallback is meant when the picture url is broken 💭 in this case it's rendered only when the URL is undefined/empty

**Pull request merged**